soc/xtensa/intel_adsp/cavs/power.c

```diff
@@ -91,5 +91,8 @@ __imr void power_init(void)
 	 */
 	sys_write32(GENO_MDIVOSEL | GENO_DIOPTOSEL, DSP_INIT_GENO);
 	sys_write32(IOPO_DMIC_FLAG | IOPO_I2SSEL_MASK, DSP_INIT_IOPO);
+
+	/* Allow LPGPDMA connection to Audio Link Hub */
+	sys_write32(DSPALHO_ASO_FLAG | DSPALHO_CSO_FLAG, ADSP_DSPALHO_ADDRESS);
```

Comment:
I think it should go here:
https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/dai/intel/alh/alh.c#L69-L89
As with GPDMA these registers have been replaced on ACE HW but have the same functions.

Reply:
Ack @tmleman  @teburd  let me update. It still a bit odd that the GPDMA driver is touching registers outside the GPDMA IP, isn't it? But I'll align with the existing approach... it'll be worse if different platforms take different approaches to tackle the same problem.

Reply:
I mean to some degree its akin to the clock and power gates. Those aren't IP specific but are very much tied to the IP, how its connected, and how its used. They are required configuration bits (like the ownership select here) to use the IP.

Reply:
> It still a bit odd that the GPDMA driver is touching registers outside the GPDMA IP, isn't it?

Generally that kind of thing screams "this should be a SOC API instead". :)

Reply:
Added this to alh.c now for cAVS2.5.